app/models/click_counter.rb

```diff
@@ -38,5 +38,7 @@ def update_click_count(url:, count:)
                         handle: 'searchgov')
   rescue ActiveRecord::RecordNotFound
     Rails.logger.error("SearchgovUrl not found for clicked URL: #{url}")
+  rescue I14yDocument::I14yDocumentError
+    Rails.logger.error("I14yDocument not found for clicked URL: #{url}")
   end
 end
```

Comment:
This may be better to be more global so other methods can use it too.  See https://medium.com/rails-ember-beyond/error-handling-in-rails-the-modular-way-9afcddd2fe1b.

Reply:
`I14yDocument::I14yDocumentError` is already a custom error class that can be used by other methods. However, I realized that ``I14yDocument::I14yDocumentError` may be raised for reasons other than a missing document, so I'm going to make this message include the specific message returned by I14y.